Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -385,6 +406,31 @@ - (void)swipeFromEdge:(UIRectEdge)edge
     [self.actor swipeFromEdge:edge];
 }
 
+- (void)activateCustomActionWithName:(NSString *)name;
+{
+    KIFUIObject *found = [[self usingCustomActionWithName:name] _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
```

Comment:
We generally wrap the body of these methods that return a `KIFUIObject` in an `@autoreleasepool` block so as to not accidentally extend the lifetime of the view object beyond this action.